docs/api/users-delete-user-by-id.md

```diff
@@ -0,0 +1,54 @@
+
+
+
+# Delete user by ID
+
+Removes the user you identify in the `id` parameter from the user resource, if the user exists.
+
+## URL
+
+```shell
+
+{server_url}/users/{id}
+```
+
+## Params
+
+| Parameter name | Type | Description |
+| -------------- | ------ | ------------ |
+| `id` | number | The record ID of the user to delete|
```

Comment:
```suggestion
| `id` | number | The record ID of the user to delete |
```
There should be a space between the text and the table delimiters.

Reply:
Done. Thanks.